### DISCUSSION TITLE
How to view existing queries?

### DISCUSSION THREAD

**Discussion by meotimdihia** [mercurius-js/mercurius#695] [Q&A] [answered]
How to expose existing queries like this: 
![msedge_2022-01-02_22-53-59](https://user-images.githubusercontent.com/300961/147881364-55237e40-2b7e-4c21-a9fc-8b4a212a74c4.png)

**Comment by mcollina** [comment, marked as answer]
where did you get that snippet?

**Reply by mcollina** [reply]
![Screenshot 2022-01-03 at 10 22 06](https://user-images.githubusercontent.com/52195/147915373-b884edcb-7369-4e18-9ef3-9ffd4e4c739c.png)


Click on the "docs" button on the right.

**Reply by meotimdihia** [reply]
@mcollina This screenshot was captured on Hasura console 😀

**Reply by meotimdihia** [reply]
@mcollina thanks. I have seen it. 🙏
![msedge_2022-01-03_16-26-53](https://user-images.githubusercontent.com/300961/147915724-7f967a2e-7014-4fc1-bcca-60060f274756.png)
